### DISCUSSION TITLE
STM32 and compatibility

### DISCUSSION THREAD

**Discussion by EnricoBonfante** [Bodmer/TFT_eSPI#1544] [Compatible displays and setup files]
Hi,
I don't understand if this Library support all of the displays mentioned in the library.
For example ILI9488 and ILI9341 can work with any STM32 of F1,F2,F4,F7 MCUs ?
I'm currently working with STM32F405rg.
Thank you for your work!

**Comment by Bodmer** [comment]
Yes it supports all those displays and it should work fine with that processor. Displays must hace a DC (Data/Command) pin, this is sometimes called RS or AO.

**Reply by EnricoBonfante** [reply]
Perfect , Thankyou!
